techniques/systemSettings/remoteAccess/sshConfiguration/4.0/main.st

```diff
@@ -144,7 +144,7 @@ bundle agent rudder_openssh_server_configuration(class_prefix, service_name, par
 
       "any"
         usebundle  => rudder_common_report("${service_name}", "result_success", "${${params}[report]}", "SSH process", "None", "The ${service_name} service is running"),
-        ifvarclass => "!${class_prefix}_down";
+        ifvarclass => "!${class_prefix}_down.!(service_restart_ssh_reached|service_restart_sshd_reached)";
```

Comment:
why reached ?
shouldn't it be:
!${class_prefix}_down.!(service_restart_ssh_ok|service_restart_sshd_ok) ?